internal/api/api.go

```diff
@@ -75,6 +78,18 @@ func (api *API) configureLogger() {
 		log.WithError(err).Warnf("failed to parse log level, falling back to %q", log.GetLevel().String())
 	}
 
+	logFormat := os.Getenv(envLogFormat)
```

Comment:
How does this work together with[ the logzio hook](https://github.com/form3tech-oss/logrus-logzio-hook/blob/master/pkg/hook/hook.go#L30)?

Reply:
AFAIK hooks are independent of general logging and fire in addition to it. 
If I read it correctly, `logrus` [README](https://github.com/sirupsen/logrus#hooks) and [source code](https://github.com/sirupsen/logrus/blob/79c5ab66aa2ce7d9ff7b3c437ebc22fcc519a967/entry.go#L271) also suggest so. In mentioned code fragment, a hook receive `Entry` struct so it's not affected by the global logger formatter.

Reply:
Have you verified the logs observed in logz.io is via logshipper and not the logrus hook? if it's the latter, would your change have any benefits?

Reply:
@sata-form3 I [double-checked](https://github.com/form3tech/tooling-team/issues/879) today. There's a logGroup attached to logs shipped from `cloudwatch` with lambda. This logGroup is present on `github-team-approver` logs on logz.io meaning they indeed come from the lambda and not the hook.